filebeat/docs/howto/migrate-to-filestream.asciidoc

```diff
@@ -1,14 +1,23 @@
 [[migrate-to-filestream]]
 == Migrate `log` input configurations to `filestream`
 
-The `filestream` input has been generally available since 7.14. So it is high time you
-migrate your existing `log` input configurations.  The `filestream` input comes with many
-improvements over the old input, such as configurable order for parsers and more.
+The `filestream` input has been generally available since 7.14. So, it is highly recommended
```

Comment:
```suggestion
The `filestream` input has been generally available since 7.14 and it is highly recommended
```

The `So, ` still makes it seem informal which is a bit weird to me in technical documentation. You removed the "high time" already but I think we can go even further :)